### ISSUE TITLE
Switch test with jdk15 openj9 to jdk16 openj9

### ISSUE THREAD

**Issue by sophia-guo** [AdoptOpenJDK/run-aqa#68]
Functional tests with openj9 jdk15 get compiling errors. We need to switch tests from openj9 jdk15 to openj9 jdk16. hotspot jdk16 may not be available yet https://github.com/AdoptOpenJDK/build-jdk/issues/21.

nightly jdk is available: 
https://api.adoptopenjdk.net/v3/binary/latest/16/ea/linux/x64/jdk/openj9/normal/adoptopenjdk
https://api.adoptopenjdk.net/v3/binary/latest/16/ea/linux/x64/jdk/hotspot/normal/adoptopenjdk

**Comment by sophia-guo** [comment]
openjdk sanity jdk_foreign_0_FAILED with both hotspot and openj9

```
Test results: passed: 35; failed: 7; error: 1
Report written to /Users/runner/work/runaqaTest/runaqaTest/jvmtest/openjdk/report/html/report.html
Results written to /Users/runner/work/runaqaTest/runaqaTest/openjdk-tests/TKG/output_16127981324885/jdk_foreign_0/work
Error: Some tests failed or other problems occurred.

jdk_foreign_0_FAILED

```

https://github.com/sophia-guo/runaqaTest/runs/1855868851?check_suite_focus=true

**Comment by sophia-guo** [comment]
Closed by #69
